docs/manifest.md

```diff
@@ -200,6 +201,11 @@ Unique identifier of the application as defined by the application creator. The
 
 URL of the Application icon shown in the Outreach Marketplace.
 
+### external installation Url
+
+URL of the external installation page. This is the optional property and is used only in case when the application should be installed outside
+of Outreach Marketplace
```

Comment:
Add sentence to the `configuration` section saying that "Configuration is ignored for applications with `externalInstallationUrl` specified

Reply:
done